### PR TITLE
Removing unused variables from lambda capture expressions

### PR DESCRIPTION
**Pull request by wayne-chen**
No test plan is required.  As long as the code compiles, the PR can be merged.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11216/HighFidelity-Beta-PR13571-bf5cdb0.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11216/HighFidelity-Beta-PR13571-bf5cdb0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11216/HighFidelity-Beta-Interface-PR13571-bf5cdb0.exe

File: interface/src/workload/GameWorkloadRenderer.cpp [hunk 1]
```diff
@@ -40,7 +40,6 @@ void GameSpaceToRender::run(const workload::WorkloadContextPointer& runContext,
     auto visible = _showAllProxies || _showAllViews;
     auto showProxies = _showAllProxies;
     auto showViews = _showAllViews;
-    auto freezeViews = _freezeViews;
```
Comment (wayne-chen) [inline interface/src/workload/GameWorkloadRenderer.cpp]: Removed variable instantiation before usage in lambda.

File: libraries/render-utils/src/Model.cpp [hunk 2]
```diff
@@ -1575,13 +1575,12 @@ void Model::removeMaterial(graphics::MaterialPointer material, const std::string
     for (auto shapeID : shapeIDs) {
         if (shapeID < _modelMeshRenderItemIDs.size()) {
             auto itemID = _modelMeshRenderItemIDs[shapeID];
-            bool visible = isVisible();
```
Comment (wayne-chen) [inline libraries/render-utils/src/Model.cpp]: And here too.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11217/HighFidelity-Beta-PR13571-8f23fb0.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11217/HighFidelity-Beta-PR13571-8f23fb0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11217/HighFidelity-Beta-Interface-PR13571-8f23fb0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11217/HighFidelity-Beta-PR13571-8f23fb0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11217/HighFidelity-Beta-Interface-PR13571-8f23fb0.exe

**Pull request merged**